packages/core/src/mergeClasses.ts

```diff
@@ -139,7 +137,11 @@ export function mergeClasses(): string {
   let atomicClassNames = reduceToClassName(resultDefinitions, dir!);
 
   // Each merge of classes generates a new sequence of atomic classes that needs to be registered
-  const newSequenceHash = hashSequence(atomicClassNames, dir!);
+  const newSequenceHash = hashSequence(
+    atomicClassNames,
+    dir!,
+    process.env.NODE_ENV === 'production' ? [] : sequencesIds,
```

Comment:
```suggestion
    sequencesIds,
```

nit: just checked this part, it seems that we can just pass it always without checks